MAINTAINERS.yml

```diff
@@ -995,8 +995,8 @@ Documentation Infrastructure:
 Release Notes:
   status: maintained
   maintainers:
-    - nashif
-    - aescolar
+    - dkalowsky
```

Comment:
Not sure who this guy is but I’m pretty sure he’s not working on zephyr 😎. dkalowsk… blame Intel